README.md

```diff
@@ -198,6 +198,27 @@ in [Endpoints.scala](https://github.com/VirtusLab/ide-probe/tree/master/core/api
 
 Note that any communication with the probe is synchronous.
 
+## Screenshots
+
```

Comment:
I'd mention **when** the screenshots are taken; also, that it only happens in Xvfb display mode

Reply:
> it only happens in Xvfb display mode

(at least I'd assume so?)